File: WeNeed1/WeNeed1.Service/Database/Review.cs
using System;
using System.Collections.Generic;

namespace WeNeed1.Service.Database;

public partial class Review
{
    public int Id { get; set; }

    public int UserId { get; set; }

    public int SportsCenterId { get; set; }

    public string? Content { get; set; }

    public decimal Rating { get; set; }

    public DateTime Created { get; set; }

    public virtual SportsCenter? SportsCenter { get; set; }
}

File: WeNeed1/MobileUI/playermobile/lib/models/review.dart
import 'package:json_annotation/json_annotation.dart';

part 'review.g.dart';

@JsonSerializable()
class Review {
  final int? id;
  final int? userId;
  final int? sportsCenterId;
  final String? content;
  final double? rating;
  final DateTime? created;

  Review({
    this.id,
    this.userId,
    this.sportsCenterId,
    this.content,
    this.rating,
    this.created,
  });

  factory Review.fromJson(Map<String, dynamic> json) => _$ReviewFromJson(json);

  Map<String, dynamic> toJson() => _$ReviewToJson(this);
}
